http/src/main/resources/reference.conf

```diff
@@ -149,6 +149,12 @@ serviceAccounts {
   }
 }
 
+pubsub {
+  pubsubGoogleProject = "broad-dsde-dev"
```

Comment:
Not sure we should have `broad-dsde-dev` in Leo reference.conf.. this should probably just be in firecloud-develop.

Reply:
Oops,  it is so I think I can remove it, it was just needed to run leo locally before that work was done